app/controllers/admin/unmatched_games_controller.rb

```diff
@@ -0,0 +1,26 @@
+# typed: true
+class Admin::UnmatchedGamesController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    authorize nil, policy_class: Admin::UnmatchedGamesPolicy
+    skip_policy_scope
+
+    @unmatched_games = Views::GroupedUnmatchedGame.where(external_service_name: 'Steam')
```

Comment:
is there a reason we care about the steam service unmatched specifically? do any other end up here if we're not applying the where?

Reply:
nope, the where is pretty much unnecessary